aten/src/ATen/core/ivalue.h

```diff
@@ -314,6 +315,20 @@ struct CAFFE2_API IValue final {
     return static_cast<at::TensorImpl*>(payload.as_intrusive_ptr);
   }
 
+  IValue(at::Storage s) : tag(Tag::Storage), is_intrusive_ptr(static_cast<bool>(s)) {
```

Comment:
Do we really need to conditionally set `is_intrusive_ptr` here? It looks to me like empty `Storage` doesn't have a specialized singleton representation; it's just `nullptr`